packages/marble-api/scripts/openapi.yaml

```diff
@@ -1657,6 +1657,78 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
+  /scenario-publications/preparation:
+    get:
+      tags:
+        - Scenario Publications
+      summary: Get the preparation status of a scenario publication
+      operationId: getScenarioPublicationPreparationStatus
+      security:
+        - bearerAuth: []
+      requestBody:
+        description: 'Describe the scenario publication action'
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - scenario_iteration_id
+              properties:
+                scenario_iteration_id:
+                  type: string
+                  format: uuid
+        required: true
+      responses:
+        '200':
+          description: The preparation status of the scenario publication
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - preparation_status
+                properties:
+                  preparation_status:
+                    type: string
+                    enum:
+                      - 'PENDING'
+                      - 'IN_PROGRESS'
+                      - 'DONE'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+    post:
+      tags:
+        - Scenario Publications
+      summary: Prepare a scenario publication
+      operationId: prepareScenarioPublication
+      security:
+        - bearerAuth: []
+      requestBody:
+        description: 'Describe the scenario publication action'
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - scenario_iteration_id
+              properties:
+                scenario_iteration_id:
+                  type: string
+                  format: uuid
+        required: true
+      responses:
+        '204':
+          description: The preparation of the scenario publication has been started
```

Comment:
Non bloquant call (we can decide to await on the creation async work if needed but IIUC this is not what we want)

Reply:
Well the go server will reply anyway (while doing work in the background).
Later we could image to do something advanced where we wait for up to N seconds and then return with a "work is continuing in the background) perhaps.

Reply:
We can change for a `202` status code, but it's not really important.